src/components/Navbar.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
+import { useCart } from "../Context/cart-context";
+import { Link } from "react-router-dom";
+import { useNavigate } from 'react-router'
+import { useAuth } from '../Context/AuthProvider'
+export function Navbar() {
+  const navigate = useNavigate()
+  const { cartCount, wishCount } = useCart();
+  const { isUserLogin } = useAuth()
+  function GetIcon() {
+    const loginStatus = JSON.parse(localStorage?.getItem("AuthDetails"));
+    const icon = loginStatus?.userIcon.toUpperCase();
+    if (icon) {
+      return icon
+    }
+    else {
+      return "_"
```

Comment:
What are we returning here?

Reply:
Code was breaking while loading avatar from localstorage so added it handle the error